src/main/java/jalasoft/com/behaviors/MemoryBehavior.java

```diff
@@ -0,0 +1,80 @@
+package jalasoft.com.behaviors;
+
+import jalasoft.com.models.Board;
+import jalasoft.com.models.CardState;
+import jalasoft.com.ui.MemoryCard;
+
+public class MemoryBehavior implements GameBehavior {
+
+  char firstSymbol = '-';
+  int firstCardId = -1;
+  char secondSymbol = '-';
+  int secondCardId = -1;
+
+  int clickCounter = 0;
+
+  @Override
+  public void performAction(Board board) {
+    clickCounter++;
+    checkPairSymbols(board);
+  }
+
+  public void checkPairSymbols(Board board) {
+    setSymbols(board);
+    checkSymbols(board);
+  }
+
+  public void setSymbols(Board board) {

```

Comment:
please update this method so you do not have duplicated code and do not use break